docs/howtos/cut-a-new-release.md

```diff
@@ -1,17 +1,20 @@
 # Application Services Release Process
 
 ## Make a new release from latest main.
+1. Validate your changes for consumers.
```

Comment:
I'm working on getting the smoke tests to pass on a local branch with nothing but these doc changes. If I can't manage to do that, I'm going to remove the smoke tests from this step as I don't think it would be fair for someone (particularly the uninitiated) to resolve issues running these tests that are unrelated to any breaking changes.